scripts/framework_to_yaml.py

```diff
@@ -0,0 +1,36 @@
+import yaml
```

Comment:
I though for the MVP (only), we might end up with demand for quick tweaks to front-end text, which it would help to be able to do through siteleaf. 

Long-run, the _data way, and potentially even handling this with an import from another repository, sounds good to me.